### DISCUSSION TITLE
Security Vulnerability Fixes v7.4.6, v7.3.13, v7.2.8

### DISCUSSION THREAD

**Discussion by guustnieuwenhuis** [MasaCMS/MasaCMS#288] [Announcements]
### What was the issue ?

We've been informed about multiple high and critical vulnerabilities in Masa CMS.

### What is fixed ?

* The mentioned vulnerabilities have been fixed.
* Additional security improvements have been made.

### What versions are affected?

Masa CMS versions 7.2, 7.3 and 7.4 are affected.

### What should you upgrade ?

* If you’re on Masa CMS 7.4, you should update to version 7.4.6 immediately. This can be done by using the “Update Masa CMS Core” option in the menu of the Masa CMS Administrator or by applying a manual update.
* If you’re on Masa CMS 7.3, you should update to version 7.3.13 immediately. This can be done by applying a manual update.
* If you’re on Masa CMS 7.2, you should update to version 7.2.8 immediately. This can be done by applying a manual update.

### What's Changed

* Disable enableMuraTag, enableDynamicContent and sharableRemoteSessions by default
* Additional security improvements have been made

Full Changelog v7.4.6: [7.4.5...7.4.6](https://github.com/MasaCMS/MasaCMS/compare/7.4.5...7.4.6)
Full Changelog v7.3.13: [7.3.12...7.3.13](https://github.com/MasaCMS/MasaCMS/compare/7.3.12...7.3.13)
Full Changelog v7.2.8: [7.2.7...7.2.8](https://github.com/MasaCMS/MasaCMS/compare/7.2.7...7.2.8)

**Comment by sebgmc** [comment]
Hi MASA CMS, upgrading to 7.4.6 gave us an error we couldn't return from. Please find attached. We've gone back to 7.4.5 for now and hope to find out why this is happening. Restarting Lucee several times did not help...
[index.txt](https://github.com/MasaCMS/MasaCMS/files/14664294/index.txt)

**Reply by sebgmc** [reply]
OK, I've narrowed it down to the following four files that generate an error on our end when upgrading from 7.4.5 to 7.4.6:

- core\mura\content\approval\approvalRequestBean.cfc
- core\mura\mailinglist\memberManager.cfc
- core\mura\user\userManager.cfc
- core\mura\user\userUtility.cfc

Restoring these three CFC's in a 7.4.6 version to the 7.4.5 version means we now run a hybrid version of MASA CMS :-)

All other files in our MASA CMS installation are now the same as 7.4.6, except there three.

Any ideas? Maybe the version of Lucee we're running?

**Comment by tbugler70** [comment]
What is the proposed work around for embedding custom modules without the Mura tag?

**Comment by mike2907** [comment]
I just upgraded the core from 7.4.3 to 7.4.6. The new version broke some javascripts that I was including. It also ignores CSS class definitions that I have included in the source. I've spent time looking through the documentation, but I haven't figured out how to allow them. Can you provide guidance on what I might do to address this problem? Thanks.
